### DISCUSSION TITLE
How to get Easter Egg Addons? (Dango Rain, etc)

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#1423] [Help]
How to access the easter egg addons on the settings page...

1. Open the settings page!
2. Do the Konami code! (<kbd>↑</kbd><kbd>↑</kbd><kbd>↓</kbd><kbd>↓</kbd><kbd>←</kbd><kbd>→</kbd><kbd>←</kbd><kbd>→</kbd><kbd>B</kbd><kbd>A</kbd>)
3. Now you have easter egg addons! Enable them and enjoy!
![image](https://user-images.githubusercontent.com/72760579/105918364-ce75ea00-6001-11eb-80a6-a83c27cdbb87.png)

Current Addons!
- Cat Blocks (But  they aren't hidden anymore)
- [Semicolon Glitch](https://en.scratch-wiki.info/wiki/Semicolon_Glitch)
- Fix capitalization of Account Settings
- Dango Rain!

**Comment by cobaltt7** [comment]
It's called an Easter egg for a reason...

And it works in the search bar
________________________________

**Reply by GrahamSH-LLK** [reply]
A. Hmm, kinda good point
B. Yeah

**Reply by TheColaber** [reply]
A. Yes, but the devs deserve to know
B. Good point, I'll remove that

**Reply by ghost** [reply]
Me: I disabled it. Yeah! I miss these blocks.
Blocks: Still Cat Blocks
Also Blocks: *meow*

**Comment by Riptide-Scratch** [comment]
If their smart enough they just look through the source code

**Reply by retronbv** [reply]
Most users will be installing through the extension webstores, they most likely do not even know it’s open source.

**Reply by Riptide-Scratch** [reply]
But it was included in the description of the extension

**Reply by scratchusernamemrtbts** [reply]
> But it was included in the description of the extension

Most people don't read the description

**Reply by TheColaber** [reply]
> But it was included in the description of the extension

*On gh, not the actual extension.

**Reply by Genius398** [reply]
> But it was included in the description of the extension

I can't see it

**Reply by 3quarterscolon** [reply]
> But it was included in the description of the extension

Most people don't/can't read the source code of their browser extensions. ScratchAddons is popular enough it has spread outside of developer circles.

**Comment by Riptide-Scratch** [comment]
I've spent to long in the scratch-wiki trying to find a secret code o_o

**Reply by pufferfish101007** [reply]
Lol scratch adfons ism't even allowed to be mentioned on the scratch wiki

**Reply by retronbv** [reply]
> Lol scratch adfons ism't even allowed to be mentioned on the scratch wiki

It’s on the French wiki

**Reply by pufferfish101007** [reply]
Ah ok makes sense the st only cares about the English wiki (insert shruggy thing here idk how to do it)

**Reply by retronbv** [reply]
here it is: `¯\\\_(ツ)\_/¯`
you have to do that or the markdown makes it weird and stuff

**Reply by Riptide-Scratch** [reply]
i mean like clues.

**Reply by cobaltt7** [reply]
the English wiki "green flag" page mentions the mute addon

Except they say it's in vanilla scratch xD

**Reply by nexensys** [reply]
OMG it is! XD

**Reply by WorldLanguages** [reply]
@RedGuy12 Keep in mind scratch 2.0 did mute the project when doing ctrl+greenflag

**Reply by mxmou** [reply]
The information about ctrl+clicking was added in 2016. I'll correct it.

**Comment by Only-a-cat** [comment]
> Tutorial on how to access the easter egg addons on the settings page...
> 
> 1. Open the settings page!
> 2. Type in the Konami code!
> 3. Now you have easter egg addons! Turn them on and enjoy!
>    ![image](https://user-images.githubusercontent.com/72760579/105918364-ce75ea00-6001-11eb-80a6-a83c27cdbb87.png)

I search and nothing appears .-.

**Reply by TheColaber** [reply]
Not search... do.

**Reply by Genius398** [reply]
> Not search... do.

Means?

**Reply by TheColaber** [reply]
up up down down left right left right b a in settings page

**Reply by 3quarterscolon** [reply]
Clarification: press the up, up, down, down, left, right, left, right, B, and A keys (in that order) while in the settings page. Don't type them into anything, just press them.

**Comment by cobaltt7** [comment]
push up up down down left right left right b a
________________________________

**Reply by Genius398** [reply]
> push up up down down left right left right b a
> […](#)
> ________________________________ From: Only-a-cat <notifications@github.com> Sent: Thursday, February 18, 2021 9:06 AM To: ScratchAddons/ScratchAddons <ScratchAddons@noreply.github.com> Cc: Paul Reid <paul@reid-family.org>; Comment <comment@noreply.github.com> Subject: Re: [ScratchAddons/ScratchAddons] How to get Easter Egg Addons? (Cat Blocks) (#1423) Tutorial on how to access the easter egg addons on the settings page... 1. Open the settings page! 2. Type in the Konami code! 3. Now you have easter egg addons! Turn them on and enjoy! [image]<https://user-images.githubusercontent.com/72760579/105918364-ce75ea00-6001-11eb-80a6-a83c27cdbb87.png> I search and nothing appears .-. — You are receiving this because you commented. Reply to this email directly, view it on GitHub<[#1423 (comment)](https://github.com/ScratchAddons/ScratchAddons/discussions/1423#discussioncomment-381291)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOT5DETOJW76VI7DF5FGMR3S7UUINANCNFSM4WUIQXXQ>.

I tried it nothing happened

**Comment by Hans5958** [comment]
I propose moving this as a wiki page, instead of making a discussion post.

EDIT: I forgot that I already made on https://scratchaddons.com/faq#how-can-you-activate-the-easter-egg-addons, but still.

**Reply by TheColaber** [reply]
i agree but nobody checks the wiki :P

**Reply by cobaltt7** [reply]
@TheColaber i do, i read practically every page on there

**Reply by TheColaber** [reply]
@RedGuy12 But "fans" don't... as well as anyone else who would probably ask to know here. Plus, it will most likely be hidden under all the other pages we have.

**Reply by Hans5958** [reply]
> i agree but nobody checks the wiki :P

I'm not sure about that.

**Reply by CloudGate1234** [reply]
> i agree but nobody checks the wiki :P

there's wiki! I didn't know that. you should make it more accessible

**Comment by Genius398** [comment]
> Tutorial on how to access the easter egg addons on the settings page...
> 
> 1. Open the settings page!
> 2. Type the Konami code!
> 3. Now you have easter egg addons! Turn them on and enjoy!
>    ![image](https://user-images.githubusercontent.com/72760579/105918364-ce75ea00-6001-11eb-80a6-a83c27cdbb87.png)

Where to type the code?

**Reply by retronbv** [reply]
The main page

**Comment by Explosion-Scratch** [comment]
I edited the main comment to include the Konami code (⬆⬆⬇⬇⬅➡⬅➡🅱🅰)

**Reply by just-a-hriday** [reply]
I suggest you use the `<kbd>` html tag to make it <kbd>↑</kbd><kbd>↑</kbd><kbd>↓</kbd><kbd>↓</kbd><kbd>←</kbd><kbd>→</kbd><kbd>←</kbd><kbd>→</kbd><kbd>B</kbd><kbd>A</kbd>.

That's
```HTML
<kbd>↑</kbd><kbd>↑</kbd><kbd>↓</kbd><kbd>↓</kbd><kbd>←</kbd><kbd>→</kbd><kbd>←</kbd><kbd>→</kbd><kbd>B</kbd><kbd>A</kbd>
```

Yes, I know that's very long code.

**Comment by Riptide-Scratch** [comment]
![image](https://user-images.githubusercontent.com/70417080/110144065-c6d9fb80-7d8c-11eb-8dc6-7c20bb5e9f73.png)
it works on turbowarp =)

**Comment by fhcirf111** [comment]
What is the semicolon glitch?

**Reply by mxmou** [reply]
https://en.scratch-wiki.info/wiki/Semicolon_Glitch

**Reply by hello-smile6** [reply]
> https://en.scratch-wiki.info/wiki/Semicolon_Glitch

Cool!

**Comment by ghost** [comment]
I play gradius. also cool easter eggs

**Comment by TheSmallyDog** [comment]
Awesome! Also the wobbly sprite dragging and the mystery mode might be cool to see added :)

**Comment by SomeCodingCoolGuy** [comment]
You could turn on Caps Lock before doing the Konami Code so it would work better.

**Reply by retronbv** [reply]
there shouldnt be any difference

**Comment by hasnain123333** [comment]
Hmm...Can we remove the easter egg addons??

**Reply by Secret-chest** [reply]
Why?

**Comment by hello-smile6** [comment]
Didn't know arrow keys were Unicode characters!

**Comment by traincrisis** [comment]
bump

**Comment by mybearworld** [comment]
Featured Dangos should probably be added to the list above (although the addon doesn't work for me, is it already disabled somehow?)

**Reply by BroJac5246** [reply]
It's not intended to work; it was April Fools'-only. A version that will work not on April Fools' is currently being developed for archival purposes.

**Reply by mybearworld** [reply]
Ah, okay, I just said that because it didn't work for me on April Fool's either.

**Reply by mybearworld** [reply]
`featured-dangos-v2` now exists.
